common/app/common/configuration.scala

```diff
@@ -20,21 +20,56 @@ import scala.util.{Failure, Success, Try}
 
 class BadConfigurationException(msg: String) extends RuntimeException(msg)
 
+case class InstallVars(stack: String, app: String, guStage: String, awsRegion: String, configBucket: String)
+
 object GuardianConfiguration extends Logging {
 
   import com.gu.cm.{Configuration => CM}
   import com.typesafe.config.Config
   lazy val configuration = {
 
-    val stage = {
+    val installVars = {
       val p = new JavaProperties()
       p.load(new FileInputStream(s"/etc/gu/install_vars"))
-      p.getProperty("STAGE", "DEV")
+      val stack = p.getProperty("stack", "frontend")
+      // if got config at app startup, we wouldn't need to configure it
+      val app = p.getProperty("app", "dev-build")
+      val stage = p.getProperty("STAGE", "DEV")
+      val region = p.getProperty("region", "eu-west-1")
+      val configBucket = p.getProperty("configBucket", "aws-frontend-store")
+      if (stage == "DEV" && new File(s"${System.getProperty("user.home")}/.gu/frontend.properties").exists) {
+        throw new RuntimeException(
+          "\n\nYou have a file ~/.gu/frontend.properties with secrets - please delete that file and any copies as it is not needed.\n  " +
+            "All secrets are now stored in AWS, not on your laptop.\n  " +
+            "Any non-secret preferences can be added to a new file ~/.gu/frontend.conf. \n" +
```

Comment:
As discussed face to face, making this less encouraging for people to override property unless it is necessary for dev purpose.